aws-ssm-opsmetadata/aws-ssm-opsmetadata.json

```diff
@@ -0,0 +1,127 @@
+{
+    "typeName": "AWS::SSM::OpsMetadata",
+    "description": "Resource Type definition for AWS::SSM::OpsMetadata",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
+    "definitions": {
+        "MetadataValueString": {
+            "$comment": "Metadata Value",
+            "type": "string",
```

Comment:
Can you please let me know if pattern validation can be added?

Reply:
Validation is taken care by the service.